aviatrix/resource_aviatrix_gateway.go

```diff
@@ -1531,6 +1545,12 @@ func resourceAviatrixGatewayRead(d *schema.ResourceData, meta interface{}) error
 			}
 			d.Set("enable_jumbo_frame", jumboFrameStatus)
 		}
+
+		detectionTime, err := client.GetTunnelDetectionTime(gw.GwName)
```

Comment:
Need to add to list_vpcs_summary instead of using an individual API

Reply:
I already spoke with Cheng and he said it wasn't a good idea to put this in list_vpcs_summary.

Reply:
Waiting on 21203 to add tunnel detection time information to list_vpcs_summary